client/my-sites/site-settings/section-security.jsx

```diff
@@ -24,7 +26,7 @@ const SiteSettingsSecurity = ( { site, translate } ) => {
 		);
 	}
 
-	if ( ! site.canManage() ) {
+	if ( ! site.canManage ) {
```

Comment:
How about using the `isJetpackSite` selector instead of `site.jetpack` here? The others should be fine, as they're computed and will be there if `.jetpack` is set.